src/views/Meetups.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <v-row class="d-flex justify-center">
+    <div class="text-center">
+      <v-btn
+        elevation="3"
+        x-large
+        class="ma-5"
+        @click="navigateToPage('/meetups')"
```

Comment:
v-btn component has "to" property please use it except programmatically navigation on click event 
https://vuetifyjs.com/en/api/v-btn/#props